sdk/sui-owned-object-pools/.eslintrc.js

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+module.exports = {
```

Comment:
All the other JS packages are covered by the root eslintrc, is there a reason this needs a separate lint config?